README.md

```diff
@@ -115,6 +115,25 @@ final dashbook = Dashbook.multiTheme(
 ```
 
 ### Visibility control properties
+A `Dashbook` instance can also support localizations (i18n) and default language
```

Comment:
This should have its own title, this doc is now inside the `Visibility control properties` section.

Please create a new section for localization and add this block under it.